main.js

```diff
@@ -0,0 +1,271 @@
+    productPrice: 49.0,
+  },
+];
+
+const div = document.createElement("div");
+const ul = document.createElement("ul");
+const listItem = document.createElement("li");
+div.append(ul);
+
+document.body.append(div);
+
+function createProductList() {
+  products.forEach((products) => {
+    const li = document.createElement("li");
+    const spanCountry = document.createElement("span");
+    const spanSert = document.createElement("span");
+
+    li.innerHTML = `${products.productName}: 
+      ${products.productData.weight}g,   
+      EXP data: ${products.productData.dateOfExpiry}, 
+      Sugar Free: ${products.productData.sugarFree} 
+      ${products.productProvider},  
+      Price: ${products.productPrice} UAH`;
+
+    if (products.productData.certificate === false) {
+      spanSert.innerHTML = " ! ";
+    }
+
+    spanCountry.innerHTML = `<img src="${products.productCountry}" alt="img" width = 20px height = 16px/>`;
+
+    li.classList.add("list");
+    spanCountry.classList.add(`country`);
+    spanSert.classList.add(`${products.productData.certificate}`);
+
+    ul.append(li);
+    li.append(spanCountry);
+    li.append(spanSert);
+  });
+}
+createProductList();
+
+const listStyle = document.getElementsByClassName("list");
+const convertedArray = Array.from(listStyle);
+for (let i = 0; i < convertedArray.length; i++) {
+  if (i % 2 === 0) {
+    convertedArray[i].classList.add("style");

```

Comment:
```suggestion
    convertedArray[i].classList.add("style");
```

Class name like `style` is not meaningful. What exact does that tell developer of that element and its styling?

Reply:
Consider using `product__info`